ska_dbi/sqsh.py

```diff
@@ -110,7 +110,7 @@ def fetch(self, query):
             cmd,
             env=cmd_env,
         )
-        outlines = stdout.decode().splitlines()
+        outlines = stdout.decode("latin-1").splitlines()
```

Comment:
Tom suggests to put the encoding up as a function option.

Reply:
Which I've done.  I'm wondering if this should actually be an attribute of the class but maybe this is just done.